deployments/common/crds/appprotect.f5.com_aplogconfs.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   preserveUnknownFields: false
   scope: Namespaced
   versions:
-    - name: v1beta1
+    - name: v1
```

Comment:
Hi @soneillf5 

Could you revert this field to v1beta1 in all AppProtect related resources? 
Note that this version is not related to the version of the CRD:
```
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
...
```
In contrast, the `v1beta1` is the version of an AppProtect resource itself.
Once App Protect folks are happy to graduate the resources from `v1beta1` to `v1`, we can graduate them.

Reply:
I've reverted the changes as you've suggested but can't get the tests to work now.

Reply:
reverted and tests are passing now